tests/test_utils.py

```diff
@@ -86,6 +86,9 @@ def assert_frames_close(actual, expected, **kwargs):
         'instead found' + expected.index.values + 'vs' + actual.index.values
 
     for col in expected.columns:
+        # if for Vensim outputs where constant values are only in the first row
+        if np.isnan(expected[col].values[1:]).all():
+            expected[col] = expected[col].values[0]
         try:
             assert_allclose(expected[col].values,
```

Comment:
I'd like to revisit our definition of "close" at some point. Have you looked at it? Do you think it's appropriate? If you think we should discuss more, can you open an issue to capture whether we think we're doing it correctly?

Reply:
I haven't look at it at all. I really don't know which precision should we ask...